packages/flutter/lib/src/material/ink_highlight.dart

```diff
@@ -40,13 +40,13 @@ class InkHighlight extends InteractiveInkFeature {
     @required RenderBox referenceBox,
     @required Color color,
     BoxShape shape = BoxShape.rectangle,
-    BorderRadius borderRadius,
+    ShapeBorder border,
```

Comment:
we can't remove borderRadius, that's a breaking change. We should support it by implying a rounded rectangle shape of the given border radius.